### ISSUE TITLE
Golem with full stack of items in hand periodically throws 63-62 items on ground 

### ISSUE THREAD

**Issue by VadersFist0501** [Azanor/thaumcraft-beta#1446]
TL;DR: golems throw 62-3 items on ground after attempting to place items in blood altar

Using a empty seal to command a golem to carry items in its hand, and using the store seal to place items in a single-item storage (e.g. blood altar), causes the golem to periodically (as in not 100% of the time) throw all its items on the ground. Some possible causes for this "bug"... the blood altar takes the full stack of items from the golem, and then rejects all but 1 item, chucking the remaining 63 onto the ground, or the golem attempts to place 64 items into the altar, but then is left with 63 in its hands, which it throws down for some reason. (basically, blood magic side, which could be prevented by inventory cap checking on the golems side, or thaumcraft side, which could also be prevented by inventory cap checking...) Dunno about this one... its a little weird. However, this can easily be prevented by using a provide/stock combination. Of course, i would prefer if the empty/store combo didnt dump stacks of stone on the ground

**Comment by Azanor** [comment]
I'm guessing the items are dropped at the source container, not the altar itself?
